charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,104 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team closely collaborates with each data contributor to:
+
+*	Working with data contributors who are generating new data types to develop a deep understanding of their data
```

Comment:
In first point, replace tab with space after the asterisk to be consistent with second two points.

Reply:
I don't see the tab. I have deleted and readded spaces. If this doesn't fix it you will need to show me what you see